app/main.py

```diff
@@ -1,6 +1,36 @@
+import os
+import requests
+
+API_KEY = os.environ.get("API_KEY")
+BASE_URL = "http://api.weatherapi.com/v1/current.json"
+CITY = "Paris"
```

Comment:
Consider making the `CITY` variable configurable via an environment variable or command-line argument to increase the flexibility of the script.